src/Features/Lsif/GeneratorTest/FoldingRangeTests.vb

```diff
@@ -64,15 +64,18 @@ using System.Linq;|}", FoldingRangeKind.Imports)>
             End Using
         End Function
 
-        Private Shared Function CreateFoldingRange(kind As FoldingRangeKind?, range As Range) As FoldingRange
-            Return New FoldingRange() With
+        Private Shared Function CreateFoldingRange(kind As String, range As Range) As FoldingRange
```

Comment:
Why this is needed? It seems the enum was changed into a record but we should still be able to use the type instead of strings.
https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/VSLanguageServerClient/commit/bfca5bd96d44579b647e4e5d40c6b94043e6d4f5?refName=refs%2Fheads%2Fdevelop&path=%2Fsrc%2Fproduct%2FProtocol%2FLanguageServer.Protocol%2FFoldingRangeKind.cs&_a=compare

Reply:
@genlu the attributes must be a constant value, or theres an error
```
BC30934	Conversion from 'FoldingRangeKind' to 'Object' cannot occur in a constant expression used as an argument to an attribute.	Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests	C:\Users\dabarbet\source\repos\roslyn\src\Features\Lsif\GeneratorTest\FoldingRangeTests.vb	39	
```